### PR TITLE
Add a very basic docker health check

### PR DESCRIPTION
**Pull request by bill88t**
What the title says.
It checks if the webserver returns HTTP 200 OK on `/login`.
It will return unhealthy is the webserver is waiting for setup though..

### CODE REVIEW

File: DockerHealthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if wget --spider -S "127.0.0.1:8080/login" 2>&1 | grep -w "200" ; then
```
Comment (zadam) [inline DockerHealthcheck.sh]: `wget` prints out all the headers and some other info. I think checking for just "200" word is not specific enough ...

**Comment by zadam** [comment]
Hi, can you elaborate on this:

> It will return unhealthy is the webserver is waiting for setup though..

I tried a clean installation and the `/login` endpoint returns 200 even before the setup is finished ...

File: DockerHealthcheck.sh (hunk 1 above)
Comment (bill88t) [reply DockerHealthcheck.sh]: alright making it `HTTP/1.1 200 OK`

**Comment by bill88t** [comment]
> I tried a clean installation and the /login endpoint returns 200 even before the setup is finished ...

running it on a new container for `127.0.0.1:8080/login` goes as follows:
image

echo $? returns the previous app's exit code.

Meanwhile for `127.0.0.1:8080/setup`:
image

The true outputs of the wget are:
image

As you can see, when the container has not finished setup, `/login` returns 302 instead of 200 OK.

File: DockerHealthcheck.sh (hunk 1 above)
Comment (bill88t) [reply DockerHealthcheck.sh]: Now it also checks for the `HTTP/1.1 200 OK` to be in the 2nd line.
It should always be at that line.

**Pull request merged**